README.md

```diff
@@ -131,6 +131,14 @@ make
 export CXXFLAGS="$CXXFLAGS -DNLIGHTNING"
 ```
 
+### C-lightning
+
+```bash
+cd modules/clightning
+make
```

Comment:
17be7040338465dc99d17a89a1cad94cdaffcd07: These steps aren't enough, it would be great to put how to fetch the submodule.